txpipe/source_selector.py

```diff
@@ -364,20 +320,27 @@ def calculate_tomography(self, pz_data, shear_data, calibrators):
 
         # and calibrate the 2D sample.
         # This calibrator refers to self.select_2d
-        calibrators[-1].add_data(data)
+        calculators[-1].add_data(data)
 
         return tomo_bin, R, counts
 
+    def compute_per_object_response(self, data):
+        # The default implementation has no per-object weight
```

Comment:
Is R response or weight?

Reply:
The response, e.g. the R_gamma matrix for metacal.

Reply:
Just saw that the comment is wrong, thanks!